files/usr/share/cinnamon/applets/window-list@cinnamon.org/applet.js

```diff
@@ -210,21 +224,26 @@ function AppMenuButton(applet, metaWindow, alert) {
 
 AppMenuButton.prototype = {
     _init: function(applet, metaWindow, alert) {
-        this.actor = new Cinnamon.GenericContainer({
-            name: 'appMenu',
-            style_class: 'window-list-item-box',
-            reactive: true,
-            can_focus: true,
-            track_hover: true });
+
+	this.actor = new Cinnamon.GenericContainer({
+	    name: 'appMenu',
+	    style_class: 'window-list-item-box',
+	    reactive: true,
+	    can_focus: true,
+	    track_hover: true });
 
         this._applet = applet;
         this.metaWindow = metaWindow;
         this.alert = alert;
 
         if (this._applet.orientation == St.Side.TOP)
             this.actor.add_style_class_name('window-list-item-box-top');
-        else
+        else if (this._applet.orientation == St.Side.BOTTOM)
             this.actor.add_style_class_name('window-list-item-box-bottom');
+        else if (this._applet.orientation == St.Side.LEFT)
+            this.actor.add_style_class_name('window-list-item-box-left');
+        else if (this._applet.orientation == St.Side.RIGHT)
+            this.actor.add_style_class_name('window-list-item-box-right');
```

Comment:
Is there some reason you can't just add a simple style class of "bottom, top, right, left" here and avoid the convoluted naming? Much easier to just use "window-list-item-box.right" instead of "window-list-item-box.window-list-item-box-right". Also stays more in line with what you typically see in gtk.